### ISSUE TITLE
Building fails on Arch Linux

### ISSUE THREAD

**Issue by MrSwedish** [FWGS/hlsdk-xash3d#175]
```[ 87/167] Linking build/cl_dll/client.so
[ 88/167] Compiling dlls/animation.cpp
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CPython::CPython()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:525: undefined reference to `vtable for CPython'
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CCrossbow::CCrossbow()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:588: undefined reference to `vtable for CCrossbow'
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CRpg::CRpg()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:674: undefined reference to `vtable for CRpg'
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CGauss::CGauss()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:735: undefined reference to `vtable for CGauss'
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CEgon::CEgon()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:782: undefined reference to `vtable for CEgon'
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CHgun::CHgun()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:848: undefined reference to `vtable for CHgun'
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CSatchel::CSatchel()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:911: undefined reference to `vtable for CSatchel'
/usr/bin/ld: cl_dll/hl/hl_weapons.cpp.0.o: in function `CSqueak::CSqueak()':
/home/[user]/dev/hlsdk-xash3d/build/../dlls/weapons.h:977: undefined reference to `vtable for CSqueak'
collect2: error: ld returned 1 exit status

Waf: Leaving directory `/home/[user]/dev/hlsdk-xash3d/build'
Build failed
 -> task in 'client' failed with exit status 1 (run with -v to display more information)
 ```

**Comment by MrSwedish** [comment]
https://gist.github.com/MrSwedish/39b9973bf55fd795f8a669ca860cc5d1 < with -v

I'm on Arch Linux

**Comment by MrSwedish** [comment]
this has been fixed in 0557b2dfe319c6640b7c65956cdb55f521dd2fb5
